cdhweb/pages/models.py

```diff
@@ -36,7 +31,6 @@
 from .blocks.article_index_block import ArticleTileBlock
 from .blocks.cdh_hosted_video import HostedVideo
 from .blocks.download_block import DownloadBlock
-from .blocks.embed_block import EmbedBlock
```

Comment:
Can we delete this version now we're not using it here?

Reply:
thanks, removed

Reply:
tyvm